index.php

```diff
@@ -153,6 +153,18 @@
 	define('PHPBB_32', true);
 }
 
+// Check if greater than 3.2
+if (!file_exists($phpbb_root_path . 'phpbb\db\driver\sqlite.' . $phpEx))
```

Comment:
Can you please revert this change. I was OK with this as it was more about choosing the right sqlite class, and is not actually about phpBB versions (also 3.2 does not have the old sqlite class either so it's a false test for greater than 3.2).

This PR should just be about the 4.0 change please.

Reply:
Not a prob. 
Just a note, '!file_exists' means if the file (old sqlite class) does NOT exist.
Therefore the test is correct IMO.

Reply:
The old sqlite class/file does not exist in 3.2.
https://github.com/phpbb/phpbb/tree/3.2.x/phpBB/phpbb/db/driver

Reply:
I know..

`if (!file_exists($phpbb_root_path . 'phpbb\db\driver\sqlite.' . $phpEx))`

IF the file does NOT exists then it is 3.3 or greater.

Reply:
Notice the `!`

Reply:
Yes. And it does NOT exist in 3.2.x just like it does NOT exist in 3.3.x. It only exists in 3.1.

Reply:
Probably should have done a `class_exists` check like I had initially suggested in the originating pull request instead of a `file_exists` check. As @VSEphpbb mentioned though, this change should be reverted from this pr.

Reply:
What's the difference between the master branch and this one?

https://github.com/phpbb/quickinstall/blob/37fa565e6872d60ed06d47c12b8340b37f1a929c/includes/functions_install.php#L143-L151

Reply:
Oky I will revert it.

Reply:
@VSEphpbb Now I am seeing what you mean :+1: